scholly/base/src/layouts/layout4/App.js

```diff
@@ -6,8 +6,17 @@ import { Switch, Route } from 'react-router-dom';
 import { Helmet } from 'react-helmet';
 import Header from './components/layout/Header';
 import getRoutes from './components/router/routes';
-import './components/centered.css';
 import { TEAM_INFO } from '../../global/data';
+import './components/centered.css';
+import '../../shared/css/style.css';
+import '../../shared/css/baseColours.css';
+
+const darkMode = true;
+if (darkMode) {
+  import('../../shared/css/darkColours.css');
+} else {
+  import('../../shared/css/lightColours.css');
+}
```

Comment:
Just a personal suggestion: since PR#223 has been approved and merging soon (I guess), I think it will be a good idea not to make any change in layout4, so you won't need to fix the conflict when that PR is deleting layout 4 entirely.

Reply:
@jfan0010 Yup makes sense. I've reverted my changes to layout 4 files.